src/modal/modal.stories.ts

```diff
@@ -60,20 +61,20 @@ class ModalStory {
 	`
 })
 class AlertModalStory {
-	@Input() modalType: string;
+	@Input() modalType: AlertModalType;
 	@Input() modalLabel: string;
 	@Input() modalTitle: string;
 	@Input() modalContent: string;
-	@Input() buttons: any;
+	@Input() buttons: Array<ModalButton>;
 
 	constructor(private modalService: ModalService) { }
 
 	openModal() {
 		this.modalService.show({
 			modalType: this.modalType,
```

Comment:
Did you miss `modalType` -> `type` or there's a reason for that? I see you check for it in `show()`.

Reply:
left it for now to show the error you can expect ... I can change it to all be updated